tests/modules/test_text_reminder.py

```diff
@@ -471,6 +472,116 @@ def test_remind_when_good_dont_remind_when_cancelled(self, r_mocked_send_text, t
         self.assertTrue("Contact is cancelled." in tr.why_not_remind_reasons())
         self.assertFalse(tr.should_remind_today())
 
+    @freeze_time(FAKE_NOW)
+    @patch("modules.text_reminder.Texter.send")
+    def test_message_object_created_upon_remind(self, mocked_send_text):
+        tr = text_reminder_object("12/6/2017") # 7 days before the 6 week appointment
+        self.assertTrue(tr.should_remind_today())
+        self.assertFalse(Message.objects.filter(contact=tr.contact, direction="Outgoing", body=tr.get_reminder_msg()))
+        tr.remind()
+        self.assertEqual(1, Message.objects.filter(contact=tr.contact, direction="Outgoing", body=tr.get_reminder_msg()).count())
+        mocked_send_text.assert_called_once_with(message=tr.get_reminder_msg(),
+                                                 phone_number="1-111-1111")
+
+    @freeze_time(FAKE_NOW)
+    @patch("modules.text_reminder.Texter.send")
+    def test_message_object_not_when_no_remind(self, mocked_send_text):
+       tr = text_reminder_object("10/7/2017") # 7 days ago
+       self.assertFalse(tr.should_remind_today())
+       self.assertFalse(Message.objects.filter(contact=tr.contact, direction="Outgoing", body=tr.get_reminder_msg()))
+       tr.remind()
+       self.assertEqual(0, Message.objects.filter(contact=tr.contact, direction="Outgoing", body=tr.get_reminder_msg()).count())
+       self.assertFalse(mocked_send_text.called)
+
+    @patch("modules.text_reminder.Texter.send")
+    def test_multiple_message_objects_created_for_multiple_reminders(self, mocked_send_text):
+        freezer = freeze_time(datetime(2017, 7, 17, 0, 0))
+        freezer.start()
```

Comment:
I think there is a `with` block / context manager one can use instead of `.start` / `.stop`.